src/main/java/school/lemon/changerequest/java/generics/pair/PairUtils.java

```diff
@@ -0,0 +1,45 @@
+package school.lemon.changerequest.java.generics.pair;
+
+public class PairUtils {
+
+    public static boolean equals(Pair<?, ?> first, Pair<?, ?> second) {
```

Comment:
What about making generics the same?

I.e. it doesn't make sense to compare a `Pair<Integer,String>` with `Pair<String,Integer`.